### PR TITLE
Updated parser label and title values

### PR DESCRIPTION
**Pull request by gnott**
Testing parser changes against proposed v2 schema for label and title values.

### CODE REVIEW

File: api-raml.sha1 [hunk 1]
```diff
@@ -1 +1 @@
-36442c5e4319ea5da8d835329654c705e5280fc9
+6db8d073fa6344662219ea81be63ee024a35197c
```
Comment (giorgiosironi) [inline api-raml.sha1]: should be superceded by new version

**Comment by giorgiosironi** [comment]
Updated api-raml, @gnott can you merge https://github.com/elifesciences/elife-tools/pull/234 if it's ready and update the `elifetools` commit in `requirements.txt` here? The other PR that will be opened automatically should be closed, we have to do the update here and keep this one green.

**Comment by giorgiosironi** [comment]
I see changes happening here with the new elife-tools now ready, assigned to @lsh-0 for review.

**Comment by gnott** [comment]
Changed the title to not say pleasedonotmerge. Please merge it when it is appropriate if everything is ready.

**Pull request merged**